src/sicks3000.cc

```diff
@@ -81,7 +81,8 @@ int SickS3000::Open()
   // Setup serial device
   if (this->serial->OpenPort2() == SERIAL_ERROR)
   {
-    ROS_ERROR("SickS3000::Open: Error Opening Serial Port");
+    ROS_ERROR_ONCE("SickS3000::Open: Error Opening Serial Port");
```

Comment:
ONCE means this won't show up multiple times if the cable is reconnected multiple times?